docs/package.json

```diff
@@ -20,7 +20,8 @@
         "lint:scss": "stylelint \"**/*.{scss,html}\"",
         "lint:fix:scss": "npm run lint:scss -- --fix",
         "build:minify-images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
-        "start": "npm-run-all build:sass build:postcss --parallel *:*:watch"
+        "start": "npm-run-all build:sass build:postcss --parallel *:*:watch",
+        "postinstall": "cd .. && npm install -f"
```

Comment:
I temporarily added postinstall so that I can check the results of this PR with netlify. I think you should probably replace this PR with another method before merging.
https://deploy-preview-18041--docs-eslint.netlify.app/rules/array-callback-return

Reply:
We can add the top-level install to the workflow. :+1:

Reply:
I removed the `postinstall` and changed the workflow.
I think we probably need to modify the netlify script, but how should I do that?

Reply:
Hmm. There is no Netlify script. It just checks out `docs` and does `npm install`. 

I wonder, could we just list `"eslint": "file:../"` in `docs/package.json` and that will work?

Reply:
> I wonder, could we just list "eslint": "file:../" in docs/package.json and that will work?

I tried that locally but it doesn't seem to work. `eslint` dependencies were not installed 😓

Reply:
Do you think putting the `netlify.toml` and adding the command config will work fine?
https://docs.netlify.com/configure-builds/file-based-configuration/#build-settings

Reply:
Let's try it. I'm out of other ideas.

Reply:
I tried adding `netlify.toml` and it seems to be working fine :tada:

https://deploy-preview-18041--docs-eslint.netlify.app/rules/array-callback-return

<img width="792" alt="image" src="https://github.com/eslint/eslint/assets/16508807/7cec2ea4-6f1a-4893-99d2-9fe66fc740aa">